GapBackfill/.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
     "azureFunctions.deploySubpath": "bin/Release/net6.0/publish",
     "azureFunctions.projectLanguage": "C#",
```

Comment:
maybe we should exclude the whole .vscode folder because it will be differently for each consumer anyways